code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -222,6 +190,39 @@
 			beakers = list()
 		update_icon(UPDATE_ICON_STATE)
 
+/obj/item/grenade/chem_grenade/screwdriver_act(mob/living/user, obj/item/I)
+	if(stage == WIRED)
+		if(length(beakers))
```

Comment:
Early return on `!length(beakers)`. Also, I think you're missing any sort of return TRUE in this proc, which I assume you meant to do.